CMakeLists.txt

```diff
@@ -31,7 +33,7 @@ add_custom_target(luacheck
 )
 
 add_custom_target(luatest
-  COMMAND ${LUATEST} -v --coverage
+  COMMAND ${LUATEST} -v --coverage --shuffle all:${seed} --seed ${seed}
```

Comment:
We can pass it once: https://github.com/tarantool/luatest/blob/f639ce4c75808166163c60b50e88a7a942b3b885/luatest/runner.lua#L137-L144.